docs/configuration.md

```diff
@@ -2,6 +2,26 @@
 
 This document describes how to configure your Aeternity node installed using a release binary for joining a public network of nodes (e.g. testnet) knowing an initial network peer to join.
 
+## User-provided configuration
+
+The `aeternity` system supports user-provided parameters via a JSON- or YAML-formatted config file.
+
+### File name and location
+The format of the config file is determined from the file extension: `.json` for JSON, or `.yaml` for YAML.
+
+The location of the file can be specified in a few different ways, in order of priority:
+1. The OS environment variable `EPOCH_CONFIG` contains a filename
+2. The Erlang/OTP environment variable `-aecore config` contains a filename
+3. A file named `epoch.{yaml,json}` exists in `${HOME}/.epoch/${sname}/`
+4. A file named `epoch.{yaml,json}` exists in `${AETERNITY_TOP}/`
+
+If all above checks fail, no user configuration is applied.
+
+`${sname}` represents the name part of the Erlang node name (the part before `'@'`).
+
+### Validation
+The contents of the config file will be validated against a [JSON-Schema](https://github.com/aeternity/aeternity/blob/master/apps/aeutils/priv/epoch_config_schema.json). If any parameters violate the schema, the node will fail to start.
```

Comment:
This configuration.md file is [included in the package](https://github.com/aeternity/aeternity/blob/50dbb47c8bcd021f663025414d34d3740ccc8617/rebar.config#L108) so I suggest to avoid references on the network when those can be satisfied locally based on the unpacked release binary. [Example](https://github.com/aeternity/aeternity/blame/50dbb47c8bcd021f663025414d34d3740ccc8617/docs/release-notes/RELEASE-NOTES-1.4.0.md#L37).

----

Schema is in package.

```
$ curl -L https://github.com/aeternity/aeternity/releases/download/v1.3.0/aeternity-1.3.0-ubuntu-x86_64.tar.gz | tar t | grep schema
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   626    0   626    0     0   1763      0 --:--:-- --:--:-- --:--:--  1768
  4 54.4M    4 2404k    0     0  1680k      0  0:00:33  0:00:01  0:00:32 2552klib/jesse-1.5.2+build.362.refb0a3bae/ebin/jesse_schema_validator.beam
 73 54.4M   73 40.0M    0     0  9257k      0  0:00:06  0:00:04  0:00:02 10.1Mlib/aeutils-1.3.0/priv/epoch_config_schema.json
lib/mnesia-4.15.3/ebin/mnesia_schema.beam
100 54.4M  100 54.4M    0     0  9959k      0  0:00:05  0:00:05 --:--:-- 12.5M
data/epoch_config_schema.json
```

Reply:
You mean `data/epoch_config_schema.json`, right?

Reply:
yes

Reply:
well - wait a minute
there are two

Reply:
You are right - any of the two copies is fine.

We have two copies:
* One from priv of otp app
* One copied https://github.com/aeternity/aeternity/blob/aa7cdc9caddc1839a825d0f6d0217395b4cddebc/rebar.config#L177-L179